### PR TITLE
update api version for pod disruption budget

### PR DESCRIPTION
**Pull request by PratikMahajan**
policy/v1beta1 PodDisruptionBudget is deprecated in v1.21+,
unavailable in v1.25+; use policy/v1 PodDisruptionBudget

### CODE REVIEW

File: dist/openshift/cincinnati.yaml [hunk 1]
```diff
@@ -201,13 +201,13 @@ objects:
...
         deploymentconfig: cincinnati
-  - apiVersion: policy/v1beta1
+  - apiVersion: policy/v1
     kind: PodDisruptionBudget
     metadata:
       name: cincinnati-pdb
       namespace: cincinnati
     spec:
-      minAvailable: 1
+      minAvailable: 3
```
Comment (LalatenduMohanty) [inline dist/openshift/cincinnati.yaml]: For disconnected environments we do not need replica count as 3. So I do not think we need to change this here.

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: dist/openshift/cincinnati.yaml (hunk 1 above)
Comment (PratikMahajan) [reply dist/openshift/cincinnati.yaml]: removed

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**